### PR TITLE
fix: verify application errors

### PR DESCRIPTION
**Pull request by emilyjablonski**
To repro, on the income step of the application, enter a currency amount but don't select a period and try to continue to the next page. On application confirmation you'll get the verify error.

### CODE REVIEW

File: sites/public/src/pages/applications/financial/income.tsx [hunk 1]
```diff
@@ -204,8 +206,7 @@ const ApplicationIncome = () => {
                 error={errors.incomePeriod}
                 errorMessage={t("errors.selectOption")}
                 register={register}
-                // TODO: Commenting out temporarily due to application issues
-                // validation={{ required: true, min: 0.01 }}
+                validation={{ required: true }}
```
Comment (emilyjablonski) [inline sites/public/src/pages/applications/financial/income.tsx:209]: This is the only difference from the core PR and should solve the issue w this step

File: sites/public/src/pages/applications/household/add-members.tsx [hunk 2]
```diff
@@ -27,14 +27,18 @@ const ApplicationAddMembers = () => {
   const { conductor, application, listing } = useFormConductor("addMembers")
   const router = useRouter()
   const currentPageSection = 2
-  const householdSize = application.householdMembers.length + 1
+  const householdSize = parseInt(application.householdMembers.length) + 1
```
Comment (ColinBuyck) [inline sites/public/src/pages/applications/household/add-members.tsx:30]: What is the benefit of adding this?

**Review by ColinBuyck — APPROVED** [review]
LGTM 🦞 just one question for my own understanding

**Comment by ColinBuyck** [comment]
Shoot, meant to leave that review on the Core PR!

File: sites/public/src/pages/applications/financial/income.tsx [hunk 3]
```diff
@@ -181,12 +184,11 @@ const ApplicationIncome = () => {
             <Field
               id="income"
               name="income"
-              // TODO: Commenting out temporarily due to application issues
-              // type="currency"
```
Comment (ludtkemorgan) [inline sites/public/src/pages/applications/financial/income.tsx:185]: Do we know what the "application issues" were here?

**Review by ludtkemorgan — APPROVED** [review]
This all looks good. Just one question

File: sites/public/src/pages/applications/financial/income.tsx (hunk 3 above)
Comment (emilyjablonski) [reply sites/public/src/pages/applications/financial/income.tsx:185]: I believe it was when we were having formatting issues w the current type of input that Kris fixed in core, but I am not 100% sure if there's something I'm missing there.

**Pull request merged**